README.md

```diff
@@ -0,0 +1,127 @@
+## Blue MoveIt
+
+### RViz-only MoveIt Demo
+
+Use `rviz_demo.launch` to bring up the two-arm MoveIt stack in RViz:
+```shell
+$ roslaunch blue_moveit_bringup rviz_demo.launch
+```
+This does not require a physical arm.
+
+### Gazebo MoveIt Demo
+
+Use `gazebo_demo.launch` to bring up the two-arm MoveIt stack in the Gazebo simulator:
+```shell
+$ roslaunch blue_moveit_bringup gazebo_demo.launch
+```
+
+This particular launch file starts a Gazebo world loaded with a table and an example pick-and-place object.
+
+Note that pick and place will crash Gazebo if the gripper actuators touch the pick-and-place object, due to how mimic joints are implemented. All other arm functionalities should work.
+
+### Physical Arm with MoveIt
+
+Use this to bring up the physical Blue arm with the basic Blue controllers and MoveIt integration!
+
+Ensure your arm is powered on and the USB dongle is connected.
+
+Note that the MoveIt bringup starts the controllers immediately. The arm will not be in gravity compensation mode and will actively try to maintain its starting position.
+
+You are advised to use one of the [helper scripts](https://github.com/berkeleyopenarms/blue_helpers) to move the arm to a position that MoveIt doesn't consider to be self-colliding. Example: `$ rosrun blue_helpers right_pickup_ready_pose_commander`
+
+**Warning:** Do not run the bringup if you've just turned on the power to the arm, as this might cause the arm to violently snap to a fully zeroed position. **Wait at least 5 seconds.**
```

Comment:
@wuphilipp don't think you wrote this, but do you know why this would happen? I don't see what waiting 5 seconds from power-on would do, since the boards don't even leave their bootloaders until the hardware interface asks them to

Reply:
Humm, that's true. I have not seen this kind of behavior.

Reply:
@methylDragon any chance you have more details on when/why this happens?

Reply:
I have no idea why it happens, but it's happened fairly rarely, and only on turning on the power, usually after just shutting down the MoveIt! and ROS stack.

I suspect it could be something on the ROS side as well, perhaps improper shutdown procedures that lead to hanging signals to the USB dongle, but I haven't done much investigating since waiting eliminates the issue mostly.